Makefile

```diff
@@ -40,6 +40,99 @@ else
 	GO_BUILD_GCFLAGS =
 endif
 
+# ====================================================================================
```

Comment:
That this is automated is super cool, but I'm wondering if we can avoid patching all our `Makefiles` everywhere like this? (It's one more thing to maintain across N repos whenever something changes.)

Instead, might we adopt the pattern used for the `Tag Public Module` action on the monorepo? https://github.com/mattermost/mattermost/actions/workflows/tag-public-module.yaml

![CleanShot 2024-06-05 at 10 06 02@2x](https://github.com/mattermost/mattermost-plugin-msteams/assets/1023171/dfca44a9-74a7-41eb-902a-b0156a15f437)

Presumably this can be more centrally managed.

Reply:
My personal preference is to use the console over the GUI whenever possible, and I recommend this approach here. This is also the practice our team follows.

Similar to my previous comment, I don't have a strong opinion on this since pushing a tag in a repository is mostly outside the scope of our work. Our main goal is to reach a consensus on pushing tags for release, thereby streamlining our process. 

We don't want to strictly control how this is done. It can be done via the GUI, manually from the console, automated and calculated via a Makefile target, or by whatever method is most convenient for a team. 

Makefile Patching is allready performed  :) :
[pr-mattermost-plugin-starter-template-200](https://github.com/mattermost/mattermost-plugin-starter-template/pull/200)
[pr-mattermost-plugin-ai-197](https://github.com/mattermost/mattermost-plugin-ai/pull/197)
[pr-mattermost-plugin-apps-487](https://github.com/mattermost/mattermost-plugin-apps/pull/487)
[pr-mattermost-plugin-bulk-invite-34](https://github.com/mattermost/mattermost-plugin-bulk-invite/pull/34)
[pr-mattermost-plugin-calls-765](https://github.com/mattermost/mattermost-plugin-calls/pull/765)
[pr-mattermost-plugin-cloud-157](https://github.com/mattermost/mattermost-plugin-cloud/pull/157)
[pr-mattermost-plugin-community-32](https://github.com/mattermost/mattermost-plugin-community/pull/32)
[pr-mattermost-plugin-demo-177](https://github.com/mattermost/mattermost-plugin-demo/pull/177)
[pr-mattermost-plugin-github-786](https://github.com/mattermost/mattermost-plugin-github/pull/786)
[pr-mattermost-plugin-gitlab-497](https://github.com/mattermost/mattermost-plugin-gitlab/pull/497)
[pr-mattermost-plugin-google-calendar-80](https://github.com/mattermost/mattermost-plugin-google-calendar/pull/80)
[pr-mattermost-plugin-jira-1083](https://github.com/mattermost/mattermost-plugin-jira/pull/1083)
[pr-mattermost-plugin-metrics-28](https://github.com/mattermost/mattermost-plugin-metrics/pull/28)
[pr-mattermost-plugin-msteams-689](https://github.com/mattermost/mattermost-plugin-msteams/pull/689)
[pr-mattermost-plugin-nps-112](https://github.com/mattermost/mattermost-plugin-nps/pull/112)
[pr-mattermost-plugin-playbooks-1913](https://github.com/mattermost/mattermost-plugin-playbooks/pull/1913)
[pr-mattermost-plugin-servicenow-195](https://github.com/mattermost/mattermost-plugin-servicenow/pull/195)
[pr-mattermost-plugin-zoom-379](https://github.com/mattermost/mattermost-plugin-zoom/pull/379)

Reply:
Thanks, @phoinixgrr! I'll reply over at https://community.mattermost.com/core/pl/ejz7p7zizbg4dpa73u8i9kdoie so centralize the conversation and loop in @mickmister and others for feedback. Appreciate the context and support in achieving all this!!

Reply:
@lieut-data Any objections merging this as is ?

Reply:
Thanks @lieut-data and @phoinixgrr for bringing the discussion more widely in channel, and glad to read you reached a consensus to move forward with the approach taken here by Akis.